<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModificar</class>
 <widget class="QDialog" name="DialogModificar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Producto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f0f0f0;
}
QLabel {
    font-size: 14px;
    font-weight: bold;
    color: #2c3e50;
}
QLineEdit {
    padding: 5px;
    border: 1px solid #bdc3c7;
    border-radius: 4px;
}
QPushButton {
    background-color: #2c3e50;
    color: white;
    border-radius: 5px;
    padding: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #34495e;
}</string>
  </property>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Editar Informaci√≥n del Producto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombreMod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCantidadMod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrecioMod">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #27ae60; /* Verde para guardar */</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #c0392b; /* Rojo para cancelar */</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>